Ppap.scala

```diff
@@ -2,7 +2,7 @@ object Main extends App{
   import scala.util.Random
   
   def ppap(pp: List[String] = Nil): Unit = pp match {
-    case "pen" :: "pineapple" :: "apple" :: "pen" :: _ => println(pp.reverse.mkString + " Ah-! Pen Pineapple Apple Pen")
+    case "pen" :: "pineapple" :: "apple" :: "pen" :: _ => println(pp.reverse.mkString + " Ah-! pen-pineapple-Apple-pen!")
     case _ => ppap((if(Random.nextBoolean) "Pen" else if(Random.nextBoolean) "Pineapple" else "apple") :: pp)
```

Comment:
Since String literals and String patterns are case-sensitive, you should rewrite as the following:

```scala
def ppap(pp: List[String] = Nil): Unit = pp match {
  case "pen" :: "pineapple" :: "apple" :: "pen" :: _ => println(pp.reverse.mkString(" ") + " Ah-! pen-pineapple-Apple-pen!")
  case _ => ppap((if(Random.nextBoolean) "pen" else if(Random.nextBoolean) "pineapple" else "apple") :: pp
)
```